lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/doi/Doi.json

```diff
@@ -16,8 +22,13 @@
 			"$ref": "org.sagebionetworks.repo.model.doi.DoiStatus",
 			"required": true
 		},
+		"url": {
+			"type": "string",
+			"description": "The URL that the DOI points to.",
+			"required": false
+		},
 		"objectId": {
-			"description": "The ID of the digital object for which this DOI is created.",
+			"description": "The ID of the digital object in Synapse for which this DOI is created.",
 			"type": "string",
 			"required": true
 		},
```

Comment:
come up with a good name to distinguish Synapse metadata from DOI metadata (like creator)

Reply:
userId
mintedOn
updatedOn
updatedBy

Reply:
mintedByUserId